lib/SlackBot.js

```diff
@@ -4,6 +4,7 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var querystring = require('querystring');
 var async = require('async');
+var Cryptr = require('cryptr');
```

Comment:
If you want something secure enough, than this approach is not the perfect method. It's just a precaution that in the end, it's not a blocker for the hacker.